src/styles/main.scss

```diff
@@ -1,3 +1,59 @@
+$calendarFont: arial;
```

Comment:
It's good practise always specify fallback fonts.
```suggestion
$calendarFont: arial, sans-serif;
```

Reply:
and I can't find where you use font in your css.

Reply:
I used it in line number 40